src/vuex/modules/session.js

```diff
@@ -160,6 +160,7 @@ export default ({ apollo }) => {
 
       dispatch(`resetSessionData`)
       commit(`setSignIn`, false)
+      console.info(`Signing user out...`)
```

Comment:
Hmm can we show a notification? This feels weird if it is silent.

Reply:
Or we do ask with a prompt:

```
By switching the network you will be signed out of your address `cosmos1234`.

Switch Network / Cancel
```

Reply:
Checkout `ActionModal.js` line 521 on how to do a prompt

Reply:
yep - this is a good solution! will alerts work on mobile?

Reply:
Possible alternative: A warning message above the network list.

Reply:
we need to test it, but I assume it works also on mobile.

Reply:
> Possible alternative: A warning message above the network list.

Could also be an option.

Reply:
I have gone down the modal route. But I think it would be cool to use a custom modal to make it look fancier & also more Lunie-like

Reply:
Do you think doing a complete ActionModal for this modal is worth it? Something like this:

![image](https://user-images.githubusercontent.com/40721795/69074789-59d41280-0a30-11ea-96a1-1e4ddc82260d.png)

Reply:
hmmm sounds like a little too much effort for the first step, if I am allowed to say that. Especially as we want to have a slightly different account flow soon.